docs/ops/generation/RandomUniform_8.md

```diff
@@ -0,0 +1,259 @@
+``` 
+
+So the resulting double is obtained as follows:
+``` 
+mantissa_h = x0 & 0xfffffu;  // upper 20 bits of mantissa
+mantissa_l = x1;             // lower 32 bits of mantissa
+mantissa = (mantissa_h << 32) | mantissa_l;
+val = ((exponent << 52) | mantissa) - 1.0,
+```
+where x0, x1 are uint32 generated random values.
+
+To obtain a value in a specified range each value is processed with the following formulas:
+
+For float values:
+\f[
+result = x * (maxval - minval) + minval,
+\f]
+where *x* is random float or double value between [0..1).
+
+For integer values:
+\f[
+result = x \mod (maxval - minval) + minval,
+\f]
+where *x* is uint32 random value.
+
+
+Example 1. *RandomUniform* output with `seed` = 150, `seed2` = 10, `output_type` = f32:
+
+``` 
+input_shape    = [ 3, 3 ]
+output  = [[0.7011236  0.30539632 0.93931055]
+          [0.9456035   0.11694777 0.50770056]
+          [0.5197197   0.22727466 0.991374  ]]
+```
+
+Example 2. *RandomUniform* output with `seed` = 80, `seed2` = 100, `output_type` = double:
+
+``` 
+input_shape    = [ 2, 2 ]
+
+minval = 2
+
+maxval = 10
+
+output  = [[5.65927959 4.23122376]
+          [2.67008206 2.36423758]]
+```
+
+Example 3. *RandomUniform* output with `seed` = 80, `seed2` = 100, `output_type` = i32:
+
+``` 
+input_shape    = [ 2, 3 ]
+
+minval = 50
+
+maxval = 100
+
+output  = [[65 70 56]
+          [59 82 92]]
+```
+
+**Attributes**:
+
+* *output_type*
+
+    * **Description**: the type of the output.
+    * **Range of values**: "i32", "i64", "f16", "f32", "f64".
+    * **Type**: string
+    * **Default value**: "f32"
+    * **Required**: *No*
+
+* *seed*
+
+    * **Description**: global seed value.
+    * **Range of values**: positive integers
+    * **Type**: `int`
+    * **Default value**: None
+    * **Required**: *Yes*
+
+* *seed2*
+
+    * **Description**: operational seed value.
+    * **Range of values**: positive integers
+    * **Type**: `int`
+    * **Default value**: None
+    * **Required**: *Yes*
+
+**Inputs**:
+
+*   **1**: `shape` - 1D tensor of type *T_SHAPE* describing output shape. **Required.**
+
+*   **2**: `minval` - Scalar with type specified by the attribute *output_type*, defines the lower bound on the range 
+    of random values to generate (inclusive). **Required.**
+
+*   **3**: `maxval` - Scalar with type specified by the attribute *output_type*, defines the upper bound on the range 
+    of random values to generate (exclusive). **Required.**
+
+
+**Outputs**:
+
+* **1**: A tensor with type specified by the attribute *output_type* and shape defined by `shape` input tensor.
+
+**Types**
+
+* *T_SHAPE*: any supported integer type.
+
+*Example 1: "minval" and "maxval" are not specified.*
+
+```xml
+<layer ... type="RandomUniform">
+    <data output_type="f32" seed="150" seed2="10"/>
+    <input>
+        <port id="0">
+            <dim>3</dim>  <!-- shape value: [2 , 3, 10] -->
+        </port>
+     </input>
```

Comment:
inputs for `minval` and `maxval` are missed

Reply:
Thanks for the comment! Corrected.